pkg/monitor/intervalcreation/node.go

```diff
@@ -434,6 +435,26 @@ func failedToDeleteCGroupsPath(nodeLocator, logLine string) monitorapi.Intervals
 	}
 }
 
+func anonymousCertConnectionError(nodeLocator, logLine string) monitorapi.Intervals {
+	if !strings.Contains(logLine, "User \"system:anonymous\"") {
+		return nil
+	}
+
+	failureTime := kubeletLogTime(logLine)
+
+	return monitorapi.Intervals{
+		{
+			Condition: monitorapi.Condition{
+				Level:   monitorapi.Error,
+				Locator: nodeLocator,
+				Message: monitorapi.ReasonedMessage("FailedToAuthenticateWithOpenShiftUser", logLine),
+			},
+			From: failureTime,
+			To:   failureTime.Add(7 * time.Minute),
```

Comment:
`1*time.Second` will still appear and let us chase these.  Since the message should be repeated regularly, I don't think there's a need for multi-minute interval.